stubs/channels/channels/db.pyi

```diff
@@ -0,0 +1,22 @@
+from asyncio import BaseEventLoop
+from collections.abc import Callable, Coroutine
+from concurrent.futures import ThreadPoolExecutor
+from typing import Any, TypeVar
+from typing_extensions import ParamSpec
+
+from asgiref.sync import SyncToAsync
+
+_P = ParamSpec("_P")
+_R = TypeVar("_R")
+
+class DatabaseSyncToAsync(SyncToAsync[_P, _R]):
+    def thread_handler(self, loop: BaseEventLoop, *args: Any, **kwargs: Any) -> Any: ...
```

Comment:
We should just copy the signature from `SyncToAsync` and annotate that properly:

```suggestion
    def thread_handler(self, loop: BaseEventLoop, exc_info: OptExcInfo, task_context, func: Callable[_P, _R], *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```

I don't know what the proper type for `task_context` is.

(`OptExcInfo` needs to be imported from `_typeshed`.)